src/test/java/org/apache/sysds/test/functions/builtin/BuiltinMiceTest.java

```diff
@@ -50,33 +50,47 @@ public void testMiceMixCP() {
 		runMiceNominalTest(mask, 1, LopProperties.ExecType.CP);
 	}
 
+//	@Test
```

Comment:
@mboehm7  as discussed, the Spark tests are commented out due to "NullPointerException" in one-hot-encoding. The error could be reproduced by running the spark tests.